### PR TITLE
Added straight ramp exit info

### PR DESCRIPTION
**Pull request by dgearhart**
@dnesbitt61 @gknisely @kevinkreiser

### CODE REVIEW

File: src/odin/maneuversbuilder.cc [hunk 1]
```diff
@@ -33,6 +33,16 @@ std::list<Maneuver> ManeuversBuilder::Produce() {
     throw std::runtime_error("Trip path does not have any nodes");
   }
 
+  // Check for a single node
+  if (trip_path_->node_size() == 1) {
+    // TODO - handle origin and destination are the same
+    throw std::runtime_error("Only one node");
```
Comment (kevinkreiser) [inline src/odin/maneuversbuilder.cc]: maybe "trip path has only one node" would be more explanitory? these execption messages are what come back from the service when they  happen (if they arent caught)

File: src/odin/narrativebuilder.cc [hunk 2]
```diff
@@ -182,18 +182,58 @@ void NarrativeBuilder::FormUturnInstruction(Maneuver& maneuver) {
 }
 
 void NarrativeBuilder::FormRampStraightInstruction(Maneuver& maneuver) {
+  // 0 = Stay straight to take the ramp
+  // 1 = branch
```
Comment (kevinkreiser) [inline src/odin/narrativebuilder.cc]: holy crap flash back

**Comment by kevinkreiser** [comment]
other than exception message, ship

File: src/odin/maneuversbuilder.cc (hunk 1 above)
Comment (dgearhart) [inline src/odin/maneuversbuilder.cc]: updated phrase for now -- it will go away in the future

**Pull request merged**